src/components/inputs/file/file.component.tsx

```diff
@@ -49,6 +49,13 @@ const File: React.FC<FormFieldInputProps> = ({ field, value, setFieldValue }) =>
     },
     [setFieldValue],
   );
+  const [clearedFiles, setClearedFiles] = useState([]);
+  const handleClearFile = (fileId) => {
+    setClearedFiles((prevClearedFiles) => [...prevClearedFiles, fileId]);
+    setFieldValue(null);
+    setImagePreview(null);
+    setDataSource(null);
+  };
```

Comment:
Can you wrap this handler wrapped in a useCallback?

Reply:
@TrevorAntony I don't think we need to maintain the state of the cleared values, resetting the other three should suffice.

```ts
const handleClearFile = useCallback(() => {
  setFieldValue(null);
  setImagePreview(null);
  setDataSource(null);
}, []);
```

Reply:
yes done this